simple-loadflow/src/main/java/com/powsybl/loadflow/simple/SimpleLimitViolationDetector.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.loadflow.simple;
+
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.iidm.network.CurrentLimits;
+import com.powsybl.security.AbstractLimitViolationDetector;
+import com.powsybl.security.LimitViolation;
+import com.powsybl.security.LimitViolationType;
+
+import java.util.function.Consumer;
+
+/**
+ * Workaround for limit detection in Mw.
+ *
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class SimpleLimitViolationDetector extends AbstractLimitViolationDetector {
```

Comment:
Maybe rename it to DcLimitViolationDetector to make it clearer.

Reply:
Done